turbinia/client_test.py

```diff
@@ -30,15 +30,18 @@
 class TestTurbiniaClient(unittest.TestCase):
   """Test Turbinia client class."""
 
-  @mock.patch('turbinia.client.task_manager.PSQTaskManager.setup')
-  def testTurbiniaClientInit(self, _):
+  @mock.patch('turbinia.client.task_manager.PSQTaskManager._backend_setup')
+  @mock.patch('turbinia.state_manager.get_state_manager')
+  def testTurbiniaClientInit(self, _, __):
     """Basic test for client."""
-    config.LoadConfig() client = TurbiniaClient()
+    config.LoadConfig()
```

Comment:
Yeah, I'm not sure how I broke that.  I think I screwed up a manual merge in one of the last PRs.  Thanks.

Reply:
ack